hive_agent/config/config.py

```diff
@@ -33,7 +33,3 @@ def save_config(self):
         """Saves the current configuration state to disk."""
         with open(self.config_path, "w") as f:
             toml.dump(self.config, f)
-
-    def get_log_level(self):
```

Comment:
can you double check that this function is being used in some places? it should be sued

Reply:
Get log level method only used while creating new HiveAgent object and I've replaced it with load_default_config method in sdk_context. It still uses old config class and get the keys the same way with looking some keys in config and if cant find the key gets default values. 

https://github.com/umutunsal/hive-agent-py/blob/4f37573ccb69671849e97344f204d8630b6772b8/hive_agent/sdk_context.py#L28